features/share/graphql/golang1.x/README.md

```diff
@@ -0,0 +1,69 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# Using Golang to Work with Our Uni
+
+We have used the [machinebox/graphql](https://github.com/machinebox/graphql) module in our example to query our **Warehouse** node.
+
+## Installation
+
+Make sure you have a working Golang enironment. Run the following command to install the `machinebox/graphql` module:
+
+```bash
+go get github.com/machinebox/graphql
+```
+
+# Querying our Uni Node
+
+[In our previous step](../creating-our-uni.md) we created a new Uni and recorded the `httpsUrl` and our `apiKey` of the **Warehouse** node. We will use this information to list out the items in our inventory. The query is defined in the file `query.go`.
```

Comment:
```suggestion
[In our previous step](../creating-our-uni.md) we created a new Uni and recorded the `httpsUrl` and our `apiKey` of the **Warehouse** node. We will use this information to list the items in our inventory. The query is defined in the file `query.go`.
```

Reply:
Is query.go created by the GraphQL resolver or does the user have to create that file. If its the latter we may want to call that out as step.

Reply:
👍